source/extensions/access_loggers/common/grpc_access_logger.h

```diff
@@ -248,13 +248,8 @@ class GrpcAccessLogger : public Detail::GrpcAccessLogger<HttpLogProto, TcpLogPro
       stats_.logs_written_.inc();
       return true;
     }
-    if (Runtime::runtimeFeatureEnabled(
-            "envoy.reloadable_features.disallow_unbounded_access_logs")) {
-      stats_.logs_dropped_.inc();
-      return false;
-    }
-    stats_.logs_written_.inc();
-    return true;
+    stats_.logs_dropped_.inc();
```

Comment:
Could you please remove the un-needed `#include "common/runtime/runtime_features.h"` (and from deps)